src/inc/msquic.h

```diff
@@ -276,6 +276,59 @@ typedef struct QUIC_NEW_CONNECTION_INFO {
     const char* ServerName;
 } QUIC_NEW_CONNECTION_INFO;
 
+#define QUIC_CIPHER_SUITE_TLS_AES_128_GCM_SHA256          0x1301
+#define QUIC_CIPHER_SUITE_TLS_AES_256_GCM_SHA384          0x1302
+#define QUIC_CIPHER_SUITE_TLS_CHACHA20_POLY1305_SHA256    0x1303
+
+#ifdef CALG_AES_128
+#define QUIC_ALG_AES_128       CALG_AES_128
+#else
+#define QUIC_ALG_AES_128       0x660E
+#endif
+#ifdef CALG_AES_256
+#define QUIC_ALG_AES_256       CALG_AES_256
+#else
+#define QUIC_ALG_AES_256       0x6610
+#endif
+#define QUIC_ALG_CHACHA20      0
+
+#ifdef CALG_SHA_256
+#define QUIC_ALG_SHA_256       CALG_SHA_256
+#else
+#define QUIC_ALG_SHA_256       0x800C
+#endif
+#ifdef CALG_SHA_384
+#define QUIC_ALG_SHA_384       CALG_SHA_384
+#else
+#define QUIC_ALG_SHA_384       0x800D
+#endif
+
+#ifdef SP_PROT_TLS1_3_SERVER
+#define QUIC_TLS1_3_SERVER      SP_PROT_TLS1_3_SERVER
+#else
+#define QUIC_TLS1_3_SERVER      0x00001000
+#endif
+#ifdef SP_PROT_TLS1_3_CLIENT
+#define QUIC_TLS1_3_CLIENT      SP_PROT_TLS1_3_CLIENT
+#else
+#define QUIC_TLS1_3_CLIENT      0x00002000
+#endif
+#define QUIC_TLS1_3             (QUIC_TLS1_3_SERVER | QUIC_TLS1_3_CLIENT)
```

Comment:
Do we _really_ need to expose the protocol as server or client? I really don't like this, and would prefer to expose just TLS1.3.

Reply:
Schannel exposes it this way. We could map it internally if that would be preferred.

Reply:
I don't particularly care how schannel maps it. :) I prefer to just map the TLS protocol used, independent of role.